src/api/1/controller-admin/load-securities/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: full.html.hbs
+title: loadSecurities
+---
+
+# loadSecurities
+
+{{{since "1.6.6"}}}
+
+Load roles, profiles and users into the storage layer.
+
+The roles, profiles and users definition follow the same structure as in the body parameter of the API:
+
+ - [createRole]({{ site_base_path }}api/1/controller-security/create-role)
+ - [createProfile]({{ site_base_path }}api/1/controller-security/create-profile)
+ - [createUser]({{ site_base_path }}api/1/controller-security/create-user)
+
+**Notes:**
+
+* The file can contain any number of roles, profiles and users.
+* If a role, profile or user already exists, it will be replaced.
+* Fixtures are loaded sequentially, first the roles, then the profiles and finally the users. If a failure occurs, Kuzzle immediately interrupts the sequence.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/admin/_loadSecurities[?refresh=wait_for]
+Method: POST
+Body:
+```
+
+```js
+{
+  "roles": {
+    "role-id": {
+      /* role definition */
+    }
+  },
+  "profiles": {
+    "profile-id": {
+      /* profile definition */
+    }
+  },
+  "users": {
+    "user-id": {
+      /* user definition */
+    }
+  }
+}
+```
+
+
+### Other protocols
+
+
+```js
+{
+  "controller": "admin",
+  "action": "loadSecurities",
+  "body": <securities>
+}
+```
+
+## Arguments
+
+### Optional:
+
+* `refresh`: if set to `wait_for`, Kuzzle will not respond until the fixtures are loaded
+
+---
+
+## Response
+
+Returns a confirmation that the command is being executed.
```

Comment:
Same remark than for loadFixtures: how do I know if a failure has occured?

Reply:
Same response: you have to wait the fixtures loading with `refresh: 'wait_for'`